Makefile

```diff
@@ -244,6 +250,9 @@ update-build: upload-build
 			"ParameterKey=ContainerPort,ParameterValue=${CONTAINER_PORT}" \
 			"ParameterKey=ListenerRulePriority,ParameterValue=${LISTENER_RULE_PRIORITY}" \
 			"ParameterKey=HealthCheckPath,ParameterValue=${HEALTH_CHECK_PATH}" \
+			"ParameterKey=SlackWebhook,ParameterValue=${SLACK_WEBHOOK}" \
```

Comment:
Not sure this actually does anything. From what I have read, lambda's pretty much have to be updated manually? I know for sure I can't get the code to update.

Reply:
So changing any of these parameters doesn't cause the Lambda's environment to be updated?

Reply:
Maybe demonstrate this to me.  I have no depth in doing Lambdas via Cfn, but it seems wrong that you can't have them update via Cfn updates.  Maybe it's dependent on the Lambda versioning stuff in some way (?).